app/repositories/CacheRepository.scala

```diff
@@ -95,13 +95,17 @@ class CacheRepository @Inject() (
       Updates.setOnInsert("createdAt", now),
       Updates.set("lastUpdated", now),
       Updates.setOnInsert("_id", Codecs.toBson(UUID.randomUUID()))
-    )
+    ) ++ Seq(
+      data.resubmittedLrn.map(Updates.set("resubmittedLrn", _))
+    ).flatten
+
```

Comment:
Can codecs not handle the optionality?

 Updates.set("resubmittedLrn", Codecs.toBson(data.resubmittedLrn))

Reply:
Done